app/views/admin/users/index.html.slim

```diff
@@ -22,4 +22,6 @@ section.section
               td= user.created_at.to_date.to_s(:db)
               td= user.email
               td= user.tz
-              td= link_to "View", admin_user_path(user.id), class: "button"
\ No newline at end of file
+              td= link_to "View", admin_user_path(user.id), class: "button"
```

Comment:
if you've already got a user instance, no need to call `id` on it, rails helpers do it internally with a bit of extra intelligence anyway